Src/BootCamp.Chapter/Checks.cs

```diff
@@ -1,39 +1,344 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
+using System.IO;
+using System.Text.RegularExpressions;
+
 
 namespace BootCamp.Chapter
 {
     // This class is used to have a freedom of design, but with tests applied.
     public static class Checks
```

Comment:
The checks class should just call an implementation of other methods. It should not be implementing those methods by itself, it's there just for testing (but not having the implementation)